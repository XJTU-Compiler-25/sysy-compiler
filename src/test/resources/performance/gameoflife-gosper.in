version https://git-lfs.github.com/spec/v1
oid sha256:5e9c5f1f791f156ec3482151470b2491fbaa0520db004d7964a7ecac0c9b5882
size 428
