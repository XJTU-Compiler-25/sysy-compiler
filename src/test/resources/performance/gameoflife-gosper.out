version https://git-lfs.github.com/spec/v1
oid sha256:0c4bc6c22966cb063c00c91bfbe0c232a0da5535a5b5f675b66a7d7fff30c090
size 422
